src/main/java/group1/mips_simulator/components/Register.java

```diff
@@ -5,6 +5,13 @@
  * the CPU to read/ write quickly.
  */
 public class Register {
+    regType type;
+    public Value value;
+
+    private enum regType{
```

Comment:
Usually type names start with a capitol letter. Not critical, but just pointing it out for future reference.

Reply:
i see, i can switch those;
So, classes, objects, types, all capitalized?

Reply:
Yes! Although in java all those three things are the same. Classes are objects which create a new type. Different languages have different style conventions. Python for example